### PR TITLE
Added custom paging templates

### PR DESCRIPTION
**Pull request by CesarDenis**

### CODE REVIEW

File: docs/_style.scss [hunk 1]
```diff
@@ -47,6 +47,29 @@ h3 {
 .slick-dots {
   margin-left: 0;
 }
+.slick-thumb {
+  bottom: -45px;
+  li {
+    width: 60px;
+    height: 45px;
+    img {
+      -webkit-filter: grayscale(100%);
```
Comment (psimyn) [inline docs/_style.scss]: autoprefixer should take care of this for you

Comment (CesarDenis) [inline docs/_style.scss]: tks @psimyn

File: docs/demos.jsx [hunk 2]
```diff
@@ -458,6 +458,34 @@ var SamplePrevArrow = React.createClass({
   }
 });
 
+var CustomPadding = React.createClass({
```
Comment (escapedcat) [inline docs/demos.jsx]: `CustomPadding`is correct here?

**Comment by escapedcat** [comment]
Also seems to be related to https://github.com/akiran/react-slick/pull/36

**Pull request merged**